ydb/_topic_reader/topic_reader_asyncio_test.py

```diff
@@ -212,21 +212,27 @@ def create_message(
             _commit_end_offset=partition_session._next_message_start_commit_offset + offset_delta,
         )
 
-    async def send_message(self, stream_reader, message: PublicMessage):
-        await self.send_batch(stream_reader, [message])
+    async def send_message(self, stream_reader, message: PublicMessage, new_batch=True):
+        await self.send_batch(stream_reader, [message], new_batch=new_batch)
 
-    async def send_batch(self, stream_reader, batch: typing.List[PublicMessage]):
+    async def send_batch(self, stream_reader, batch: typing.List[PublicMessage], new_batch=True):
         if len(batch) == 0:
             return
 
         first_message = batch[0]
         for message in batch:
             assert message._partition_session is first_message._partition_session
 
+        partition_session_id = first_message._partition_session.id
+
         def batch_count():
             return len(stream_reader._message_batches)
 
+        def batch_size():
+            return len(stream_reader._message_batches[partition_session_id].messages)
+
         initial_batches = batch_count()
+        initial_batch_size = batch_size() if not new_batch else 0
```

Comment:
what about invert condition?

```suggestion
        initial_batch_size = 0 if new_batch else batch_size()
```